commands.txt

```diff
@@ -3,7 +3,7 @@ if [ ! -f sonar-project.properties ] && [ -z $SD_SONAR_OPTS ]; then echo "ERROR
 export SD_CURL_CMD_WRAPPER="sd-step exec --pkg-version 7.54.1 core/curl"
 export SD_UNZIP_CMD_WRAPPER="sd-step exec --pkg-version 6.0 core/unzip"
 SONAR_TOKEN=`$SD_CURL_CMD_WRAPPER "curl -H \"Authorization: Bearer $SD_TOKEN\" $SD_SONAR_AUTH_URL"`
-SCANNER_OPTION="\"-Dsonar.host.url=$SD_SONAR_HOST\" \"-Dsonar.login=$SONAR_TOKEN\" \"-Dsonar.projectKey=job:$SD_JOB_ID\" \"-Dsonar.projectName=$SD_PIPELINE_NAME:$SD_JOB_NAME\" \"-Dsonar.projectVersion=$SD_BUILD_SHA\" \"-Dsonar.links.scm=$SCM_URL\" \"-Dsonar.links.ci=$SD_UI_URL/pipelines/$SD_PIPELINE_ID\" $SD_SONAR_OPTS"
+SCANNER_OPTION="\"-Dsonar.host.url=$SD_SONAR_HOST\" \"-Dsonar.login=$SONAR_TOKEN\" \"-Dsonar.projectKey=job:$SD_JOB_ID\" \"-Dsonar.projectName=$SD_PIPELINE_NAME:$SD_JOB_NAME\" \"-Dsonar.links.scm=$SCM_URL\" \"-Dsonar.links.ci=$SD_UI_URL/pipelines/$SD_PIPELINE_ID\" $SD_SONAR_OPTS"
```

Comment:
what's the impact of removing this ? Did we add this for any specific reason ?

Reply:
i replied to catto in the issue for this. The project version is just something nice to tag each event.
can take a look at here: https://sonar.screwdriver.cd/project/activity?id=job%3A9550

https://github.com/screwdriver-cd/screwdriver/issues/1579